processor/src/debug.rs

```diff
@@ -74,15 +74,15 @@ impl VmStateIterator {
             (
                 &assembly_ops[self.asmop_idx - 1],
                 // difference between current clock cycle and start clock cycle of the current asmop
-                (self.clk - assembly_ops[self.asmop_idx - 1].0) as u8,
+                (self.clk as usize - assembly_ops[self.asmop_idx - 1].0) as u8,
```

Comment:
makes sense. I didn't know whether `assembly_ops[self.asmop_idx - 1].0 as u32` can outsize `u32`. But if we would have a problem anyway I guess, because then also the `u8` casting would be wrong.